library/std/src/error/tests.rs

```diff
@@ -114,6 +114,9 @@ fn error_with_no_sources_formats_multi_line_correctly() {
 }
 
 #[test]
+// NOTE(ignore/aarch64/nto) this and the next ignored test are executed as a
```

Comment:
I'd prefer to note both tests individually since someone might go add a test in between these two.